COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/CoordinateSystem.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class CoordinateSystem
+    {
+        public List<Point> listOfPoints = new List<Point>(); //List with all Points
+
+        public List<double> listOfDistsPoints = new List<double>(); //List with all Distances between the points
+
+        public void Add(Point point)
+        {
+            listOfPoints.Add(point);
+        }
+
+        public void DisplayAllPoints()
+        {
+            int count = 1;
+
+            foreach (Point i in listOfPoints)
+            {
+                Console.Write($"point {count++} : {i};\n");
+            }
+        }
+
+        public void FindDistance()
+        {
+            foreach (Point k in listOfPoints) // TODO потрібно створити якусь копію ліста, звідки видаляти член K після того, як він уже був прогнаний в циклі. Аби не дублювати ті самі відстані
+            {                                 //мб замість ліста тут краще використати черги/стаки???
+                foreach (Point l in listOfPoints)
```

Comment:
To not calculate distance between points twice (p1 to p2 and p2 to p1) you have to use 'for' loop with next conditions
Please see how I initialize second loop 'j = i + 1'

```csharp
for(var i = 0; i < listOfPoints.Count; ++i)
{
    for(var j = i + 1; j < listOfPoints.Count; ++j)
    {
        //your code
    }
}
```

Reply:
oh.... great example. TY. next time will try to do like you

Reply:
whole logic was remaden so the comment is not actual....